### PR TITLE
Nested single day events always shown only with time

### PR DESCRIPTION
**Pull request by dzonidoo**
STT-93

## Front-end checklist

- [x] This pull request is adding missing TypeScript types to modified code segments where it's easy to do so with confidence
- [x] This pull request is using TypeScript interfaces instead of prop-types and updates usages where it's quick to do so
- [x] This pull request is using `memo` or `PureComponent` to define new React components (and updates existing usages in modified code segments)
- [x] This pull request is replacing `lodash.get` with optional chaining and nullish coalescing for modified code segments
- [x] This pull request is not importing anything from client-core directly (use `superdeskApi`)
- [x] This pull request is importing UI components from `superdesk-ui-framework` and `superdeskApi` when possible instead of using ones defined in this repository.
- [x] This pull request is not using `planningApi` where it is possible to use `superdeskApi`
- [x] This pull request is not adding redux based modals
- [x] In this pull request, properties of redux state are not being passed as props to components; instead, we connect it to the component that needs them. Except components where using a react key is required - do not connect those due to performance reasons.
- [x] This pull request is not adding redux actions that do not modify state (e.g. only calling angular services; those should be moved to `planningApi`)

### CODE REVIEW

File: client/utils/events.ts [hunk 1]
```diff
@@ -1361,7 +1361,7 @@ const self = {
     canUpdateEventTime,
     canConvertToRecurringEvent,
     canUpdateEventRepetitions,
-    isEventSameDay,
+    isSameDay,
```
Comment (tomaskikutis) [inline client/utils/events.ts]: move it out of event utils since it's generic

Comment (dzonidoo) [reply client/utils/events.ts]: and put it in some helpers?

Comment (tomaskikutis) [reply client/utils/events.ts]: yep

File: client/validators/events.ts [hunk 2]
```diff
@@ -7,6 +7,7 @@ import {gettext, eventUtils} from '../utils';
 import * as selectors from '../selectors';
 import {formProfile} from './profile';
 import {PRIVILEGES, EVENTS, TO_BE_CONFIRMED_FIELD} from '../constants';
+import {isSameDay} from 'helpers';
```
Comment (tomaskikutis) [inline client/validators/events.ts]: use relative imports (starting with a dot)

**Review by tomaskikutis — APPROVED** [review]

**Pull request merged**